lib/core/src/Cardano/Wallet/DB/Store/Wallets/Model.hs

```diff
@@ -90,3 +126,79 @@ linkedTransactions (TxMetaHistory m) = Map.keysSet m
 
 walletsLinkedTransactions :: Map W.WalletId TxMetaHistory -> Set TxId
 walletsLinkedTransactions = Set.unions . toList .  fmap linkedTransactions
+
+mkTransactionInfo :: Monad m
+    => TimeInterpreter m
+    -> W.BlockHeader
+    -> TxRelationF 'With
+    -> DB.TxMeta
+    -> m WT.TransactionInfo
+mkTransactionInfo ti tip TxRelationF {..} DB.TxMeta {..} = do
```

Comment:
```suggestion
mkTransactionInfo ti tip TxRelationF{..} DB.TxMeta{..} = do
```

Reply:
Style: I prefer to put the curly braces directly after the constructor — in this way, I can distinguish function arguments by whitespace. (For example, an unattentive glance at the old line may have misled me into thinking that the function has six arguments, two of them records `{}`).

Reply:
Once we have an auto-formatter it will not be an option/I prefer :-). I must admit that you guys are really looking into these details. One day I will as well (maybe)!